pkg/cmd/provisioning/ibmcloud/service_id.go

```diff
@@ -234,8 +236,13 @@ func (s *ServiceID) UnDo(targetDir string) error {
 	}
 
 	secretFileName := filepath.Join(targetDir, manifestsDirName, fmt.Sprintf(secretFileNamePattern, s.cr.Spec.SecretRef.Namespace, s.cr.Spec.SecretRef.Name))
+	fmt.Printf("secretFileName: %s\n", secretFileName)
```

Comment:
should this be a log message, or just removed entirely?

Reply:
Will remove this line..